packages/@webex/plugin-cc/src/constants.ts

```diff
@@ -1,4 +1,9 @@
 export const POST_AUTH = 'postauth';
 export const WCC_API_GATEWAY = 'wcc-api-gateway';
 export const EVENT = 'event';
+export const READY = 'ready';
 export const SUBSCRIBE_API = 'v1/notification/subscribe';
+export const REGISTER_TIMEOUT = 20000;
+export enum CC_EVENTS {
```

Comment:
Use the new format to define enums. Please refer to how it was added for HTTP_METHODS